starter-code/routes/auth.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const passport = require('passport');
+
+const User = require('../models/user.js');
+
+const router = express.Router();
+
+router.get('/auth/facebook', passport.authenticate('facebook'));
+
+router.get('/auth/facebook/callback', passport.authenticate('facebook', {
```

Comment:
This is fine, but you don't have defined serializeUser, neither deserializeUser.